ui/app/AppLayouts/Wallet/components/AddAccountWithSeed.qml

```diff
@@ -92,6 +92,8 @@ ModalPopup {
         id: accountColorInput
         anchors.top: accountNameInput.bottom
         anchors.topMargin: marginBetweenInputs
+        anchors.left: parent.left
+        anchors.right: parent.right
```

Comment:
If most `Select`s need those anchors, it's worth it to put it in the Component directly. I think that's what I did with Input too

Reply:
IMO, anchors should not be included in a component that is reusable because they cannot be changed by the parent component. I ran in to this issue with the Select. *Most* components need a full width selector, but not all (the AssetSelector), so I really had no choice here.

Reply:
>  IMO, anchors should not be included in a component that is reusable because they cannot be changed by the parent component. I ran in to this issue with the Select

Weird, I never had that issue. Usually, if the anchors are at the top level of the component, they can be overwritten. Maybe you had one with `anchors.fill` or a `center` one. I know QML shows a warning if you mix side anchors with center anchors. But worst case, I think you can just overwrite the one you don't want with `anchors.whatever: undefined`

Anyway, it's no big deal

Reply:
Yes, you're right, you can overwrite anchors from the parent. I've changed this back as you suggested. The specific case I ran in to was a bit different. Apologies for not clarifying earlier. The case where anchors can't be overwritten is when attempting to overwrite them from an aliased property, like so:

```qml
AssetSelector {
    select.anchors.left = parent.left // causes error
}
Select {
    property alias select: inputRectangle
    Rectangle {
         id: inputRectangle
    }
}
```